src/main/java/org/springframework/cloud/deployer/spi/kubernetes/AbstractKubernetesDeployer.java

```diff
@@ -90,7 +91,18 @@ protected AppStatus buildAppStatus(KubernetesDeployerProperties properties, Stri
 	 * @param request    The deployment properties.
 	 */
 	protected Map<String, Quantity> deduceResourceLimits(KubernetesDeployerProperties properties, AppDeploymentRequest request) {
+		String memDeployer = getCommonDeployerMemory(request);
 		String memOverride = request.getDeploymentProperties().get("spring.cloud.deployer.kubernetes.memory");
```

Comment:
Why is the old `spring.cloud.deployer.kubernetes.memory` property still supported? I thought the point of introducing a common property was that all implementations would use it